Sources/TuistKit/Services/ProjectAutomationModels+FromGraphModels.swift

```diff
@@ -0,0 +1,264 @@
+import Foundation
```

Comment:
nit: maybe I would call this file ProjectAutomation+ManifestMapper or similar for consistency with other similar files (even though those are mapping between projectdescription and tuistgraph

Reply:
will rename, this was only a suggestion from @pepicrft